src/platform_impl/windows/event_loop.rs

```diff
@@ -1002,11 +1013,11 @@ unsafe fn public_window_callback_inner<T: 'static>(
         }
 
         // WM_MOVE supplies client area positions, so we send Moved here instead.
-        winuser::WM_WINDOWPOSCHANGED => {
+        WM_WINDOWPOSCHANGED => {
             use crate::event::WindowEvent::Moved;
 
-            let windowpos = lparam as *const winuser::WINDOWPOS;
-            if (*windowpos).flags & winuser::SWP_NOMOVE != winuser::SWP_NOMOVE {
+            let windowpos = lparam as *const WINDOWPOS;
+            if (*windowpos).flags & SWP_NOMOVE != SWP_NOMOVE {
```

Comment:
Yes I found many of these using some regex magic. But I finally thought that I only want to port to windows-rs in this PR since doing anything more would just make the diff and therefore reviewing harder...